app/util/__test__/originFromUrlString.test.ts

```diff
@@ -0,0 +1,14 @@
+import { originFromUrlString } from "../originFromUrlString";
```

Comment:
Sorry, I tried to get this globally adjusted within VSCode, without any success... 
```suggestion
import { it } from "vitest";
import { originFromUrlString } from "../originFromUrlString";
```